src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -45,7 +38,8 @@ protected ActionResult execute() {
         gateKeeper.verifyAccessible(
                 instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
 
-        FeedbackSessionAttributes fs = extractFeedbackSessionData();
+        InstructorFeedbackAddAction newSession = new InstructorFeedbackAddAction();
+        FeedbackSessionAttributes fs = newSession.extractFeedbackSessionData();
```

Comment:
You do not need to create a new object for this, you can simply call the method directly. Other classes (eg `InstructorCourseInstructorAddAction`) do this just fine.